src/frontend/hooks/useSjekklikheter.ts

```diff
@@ -0,0 +1,30 @@
+import { useState, useCallback } from 'react';
+import { useHttp } from '@navikt/familie-http';
+import { type Ressurs, RessursStatus } from '@navikt/familie-typer';
+
+const useSjekkLikhetPerioder = (behandlingId: string) => {
+    const [result, setResult] = useState<Ressurs<boolean>>({ status: RessursStatus.IKKE_HENTET });
+    const { request } = useHttp();
+
+    const hentSjekkLikhetPerioder = useCallback(() => {
+        setResult({ status: RessursStatus.HENTER });
+
+        request<void, boolean>({
```

Comment:
Gir boolean mening her - eller bør vi ha en enum?
`LIKE_PERIODER, EN_PERIODE, ULIKE_PERIODER` eller noe? 

For har du bare én periode - hva tenker vi at denne returnerer?

Reply:
Likhetsjekken tenkte vi kanskje bare skulle bestemme om det skulle vises en knapp ala "slå sammen like perioder". Det er et godt poeng at den bare returnerer true hvis det er flere perioder, og at det faktisk er noen perioder å slå sammen i brevet. Sånn sett tenker vi egentlig ikke at enum er vits?

Reply:
Skjønner - bare ikke så lett å komme inn senere og forstå hvordan dét funker :)